src/hooks/mongoose_hooks.erl

```diff
@@ -419,19 +412,6 @@ user_ping_response(HostType, Acc, JID, Response, TDelta) ->
     Params = #{jid => JID, response => Response, time_delta => TDelta},
     run_hook_for_host_type(user_ping_response, HostType, Acc, Params).
 
-%%% @doc The `vcard_set' hook is called to inform that the vcard
```

Comment:
it is pretty useful when we need to notify about need to download vcard again (if avatar is stored there).